### PR TITLE
Fix: wallet select UI bugs

### PR DESCRIPTION
**Pull request by egor0798**
## Changes:
- Removed extra divider for multisig staking operations confrimation screen
- Close wallet select after wallet selected
- Fixed wallet select z-index

### CODE REVIEW

File: src/renderer/features/wallets/WalletSelect/ui/WalletPanel.tsx [hunk 1]
```diff
@@ -1,19 +1,30 @@
...
 
+  const activeWallet = useUnit(walletModel.$activeWallet);
+  const [initialWallet] = useState(activeWallet);
+
+  useEffect(() => {
+    if (activeWallet?.id !== initialWallet?.id) {
+      onClose();
+    }
+  }, [activeWallet]);
+
```
Comment (egor0798) [inline src/renderer/features/wallets/WalletSelect/ui/WalletPanel.tsx:27]: Tried passing onClose prop to WalletGroup component where select event occurs but closing popup right after select event fired leads to wallet not changin after select

**Comment by egor0798** [comment]
Task linked: [CU-869307t6u Wallet management. Does not close management, after switch wallets.](https://app.clickup.com/t/869307t6u)

**Comment by egor0798** [comment]
Task linked: [CU-8693090n6 Wallet Management menu. UI overlapping](https://app.clickup.com/t/8693090n6)

File: src/renderer/pages/Operations/components/Details.tsx [hunk 2]
```diff
@@ -62,6 +62,11 @@ const Details = ({ tx, account, connection, signatory }: Props) => {
 
   const hasSender = isXcmTransaction(tx.transaction) || isTransferTransaction(tx.transaction);
 
+  const isDidverVisible =
```
Comment (Asmadek) [inline src/renderer/pages/Operations/components/Details.tsx:65]: Maybe name it little bit specifier ?  but condition looks widely too

**Review by Asmadek — APPROVED** [review]

**Comment by egor0798** [comment]
Task linked: [CU-86930934z Reject Operation. Confirm page - empty container.](https://app.clickup.com/t/86930934z)

File: src/renderer/pages/Operations/components/Details.tsx (hunk 2 above)
Comment (egor0798) [reply src/renderer/pages/Operations/components/Details.tsx:65]: Thought about that but can't find right words to describe that condition

Comment (Asmadek) [reply src/renderer/pages/Operations/components/Details.tsx:65]: IN that case we can leave it as is

**Review by pgolovkin — APPROVED** [review]

**Pull request merged**